packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1047,46 +1049,6 @@ class VM extends ServiceObjectOwner {
     return invokeRpcRaw('_createDevFS', params: <String, dynamic>{'fsName': fsName});
   }
 
-  /// List the development file system son the device.
-  Future<List<String>> listDevFS() async {
```

Comment:
We don't use the jsonrpc based devFS api anymore, because base64 encoding large files is too slow. Instead we use the HTTP API